plugins/shorewall/shorewall_log

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/python3
+# -*- python -*-
+
+"""
+
+=head1 INTRODUCTION
```

Comment:
Please use the section name `NAME` instead. This will improve the visualization in the [plugin gallery](https://gallery.munin-monitoring.org/).

Reply:
done

Reply:
Oops, I changed the `=head1` to `=name` instead of the `INTRODUCTION`, fixed now.